Explorer/Assets/Scripts/ECS/StreamableLoading/GLTF/LoadGLTFSystem.cs

```diff
@@ -29,7 +33,7 @@ internal LoadGLTFSystem(
             IPartitionComponent partitionComponent) : base(world, cache)
         {
             this.sceneData = sceneData;
-            gltfDownloadProvider = new GltFastDownloadProvider(World, sceneData, partitionComponent);
+            this.partitionComponent = partitionComponent;
```

Comment:
`IPartitionComponent` should be part of an entity with a visual representation, as its used to determine distance to player and when it should load. The injection here doesnt seem right, what was the intention?

Reply:
Uhmm you are right, since `LoadGLTFSystem` is the counterpart of `LoadAssetBundleSystem` I checked that other one and saw that the `IPartitionComponent` that's being used is the one received from the interface's `FlowInternalAsync()` so I replaced the one feeded to the asset promise for that one and removed the constructor injection, it didn't make sense.

Tackled at: https://github.com/decentraland/unity-explorer/pull/1670/commits/02568d86fc23d8677cd53af20cc90b11d3cc680e